code/modules/mob/living/carbon/xenomorph/abilities/crusher/crusher_powers.dm

```diff
@@ -106,6 +106,57 @@
 	..()
 	return
 
+/datum/action/xeno_action/onclick/crusher_stomp/charger/use_ability(atom/A)
+	var/mob/living/carbon/Xenomorph/X = owner
+	var/mob/living/carbon/V
```

Comment:
You need more descriptive variable names here. What is V supposed to be? You're making life harder for future devs